services/ui-src/src/types/report.ts

```diff
@@ -79,6 +79,7 @@ export interface MeasurePageTemplate extends FormPageTemplate {
   required?: boolean;
   stratified?: boolean;
   optional?: boolean;
+  substitutable?: boolean;
```

Comment:
couldn't think of a better word for measures that have the option to be substituted. open to ideas here

Reply:
replaceable? exchangeable? tbh substitutable is fine :)